scripts/userstyles.yml

```diff
@@ -760,6 +760,9 @@ userstyles:
     color: blue
     readme:
       app-link: "https://app.raindrop.io"
+      usage: |+
+        > [!NOTE]
+        > Set Raindrop's built-in theme to one of the first 3 themes. the 4th theme ('sunny') won't be different from the light theme
```

Comment:
As I mentioned briefly earlier, if there aren't any problems theming "sunny" with the existing CSS I don't believe this note is necessary since the experience will be the same either way.